gradient/api_sdk/clients/notebook_client.py

```diff
@@ -54,6 +59,7 @@ def create(
             container_user=container_user,
             shutdown_timeout=shutdown_timeout,
             is_preemptible=is_preemptible,
+            machine_type=machine_type,
```

Comment:
In the CLI options vm_type_label was renamed to machine_type. Why is it left here if it's not used?